model/src/pyrenew/latent/hospitaladmissions.py

```diff
@@ -162,7 +162,7 @@ def sample(
 
         Parameters
         ----------
-        latent : ArrayLike
+        latent : ArrayLike or SampledValue
```

Comment:
```suggestion
        latent_infections : ArrayLike
```
As the code written, this cannot be a `SampledValue`, since we do not call `.value` on it. And I think that's at least arguably correct

Reply:
Good catch.